library/src/main/java/com/rmielnik/list/RecyclerItem.kt

```diff
@@ -0,0 +1,20 @@
+package com.rmielnik.list
+
+import androidx.annotation.LayoutRes
+
+/**
+ * Represents an item that will be displayed a RecyclerView.
+ */
+interface RecyclerItem {
+
+    /**
+     * Identifier of the item.
+     */
+    val id: Long
```

Comment:
Important one! When you use diffutil, then recycler doesn't use stable ids. That means you can use Any as an id. Use case is that items very often has String or uuid instead of long as an id

Reply:
Good one! I'll change it to Any. Thanks